src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,66 @@
+import { Tabs as MuiTabs, TabsProps as MuiTabsProps, Tab, Box, Typography} from '@mui/material'
+import React from 'react';
+/**
+ * Tabs organize and allow navigation between groups of content that are related and at the same level of hierarchy.
+ */
+
+type TabsProps = {
+
+} & MuiTabsProps
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+function CustomTabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+  };
+}
+
+export const Tabs = ({
+  
+}: TabsProps) => {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.SyntheticEvent, newValue: number) => {
+    setValue(newValue);
+  };
+
+  return (
+    <Box sx={{ width: '100%' }}>
```

Comment:
Will this allow me to pass in new props and use the component how I would like to use it or will it always have the 3 tabs?

Reply:
@Kush-Kurense I think what Cody was trying to say was can we have it so we can pass in the three tab names as a prob?